events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+* Joining the appropriate Slack channels for KCS staff collaboration and for sending comms about KCS to the community
+* Working with the Comms Lead and Contributor Comms Subproject to get access to tooling for posting on Kubernetes Contributor-focused social media channels.
+* Writing email comms as instructed by the KCS Comms Lead.
+* Writing social media comms as instructed by the KCS Comms Lead.
+* [optional] Reviewing comms written by other volunteers, shadows, or the lead as directed by the KCS Comms Lead
+
+## KCS Website Manager
+The KCS Website Manager is expected to collaborate with all KCS Staff leads and their teams to understand what changes need to be made to the website, when those changes need to go live, and ensuring that all KCS-related website changes are planned, prioritzed, and completed.
+
+### Requirements
+- Review this document.
+- [Kubernetes Org Membership]
+- Commit to supporting KCS Comms through the current cycle and the following cycle as emeritus Website Manager (1 year, 2 cycles)
+- Basic familiarity with the [Kubernetes Contributor Site (k8s.dev)](https://github.com/kubernetes/contributor-site), especially the events section (k8s.dev/events)
+- Basic familiarity with the [Hugo website framework](https://gohugo.io/)
+- Basic familiarity with the [Markdown](https://www.markdownguide.org/) markup language
+
+### Tasks
+During the course of their KCS cycle as a website manager, KCS Website Managers are expected to complete tasks including but not limited to:
+- Joining the appropriate Slack channel(s) for KCS staff collaboration
+- Creating the page for the specific event within k8s.dev/events
+- Adding and maintaining information and pages throughout the event cycle, including:
+  - Registration info
+  - CFP & schedule info
+  - Social venue and registration info
+  - and more
+
+## Emeritus KCS Comms Lead Overview
+After completing a cycle as a KCS Comms Lead, the former lead takes on emeritus status. As an Emeritus KCS Lead, the former lead is expected to mentor the following KCS Lead(s). The Emeritus Lead can provide mentorship for as long as they like, but it's especially valuable for the previous lead to mentor their direct successor.
+
+### Requirements
+- Has completed at least 1 cycle as a KCS Comms Lead
+- [Kubernetes Org Membership]
+
+### Tasks
+- Provide mentorship to the KCS Comms Lead
+
+## Comms Rules and Guidelines
+
+### Social Media Rules
+- Send KCS comms to k8scontributors accounts on whatever platforms they exist on. At time of writing, that’s “X” and Mastodon. Social media should be sent only to contributor-facing accounts.
+  - Kubernetesio on X and the Kubernetes Linkedin accounts should not be used because their audiences include end users and the general public..
+- Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.
```

Comment:
I would again stay away from specifics here and replace it with a more general guideline of keeping the doc consistent. As screens increase pixel density, 11 pt might not be big enough.